frontend/src/metabase/query_builder/components/dataref/ModelPane/ModelPane.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { connect } from "react-redux";
+import { t, jt } from "ttag";
+import PropTypes from "prop-types";
+
+import DateTime from "metabase/components/DateTime";
+import {
+  Description,
+  EmptyDescription,
+} from "metabase/components/MetadataInfo/MetadataInfo.styled";
+import { getQuestionFromCard } from "metabase/query_builder/selectors";
+import type { Card } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+import Question from "metabase-lib/lib/Question";
+import FieldList from "../FieldList";
+import {
+  ModelPaneDetail,
+  ModelPaneDetailLink,
+  ModelPaneDetailLinkText,
+  ModelPaneDetailText,
+  ModelPaneIcon,
+  ModelPaneDescription,
+} from "./ModelPane.styled";
+
+interface ModelPaneProps {
+  show: (type: string, item: unknown) => void;
+  model: Card;
+  question: Question;
+}
+
+const mapStateToProps = (state: State, props: ModelPaneProps) => ({
+  question: getQuestionFromCard(state, props.model),
+});
+
+const ModelPane = ({ show, question }: ModelPaneProps) => {
+  const table = question.table();
+  return (
+    <div>
+      <ModelPaneDescription>
+        {question.description() ? (
+          <Description>{question.description()}</Description>
+        ) : (
+          <EmptyDescription>{t`No description`}</EmptyDescription>
+        )}
+      </ModelPaneDescription>
+      <ModelPaneDetail>
+        <ModelPaneDetailLink
+          href={question.getUrl()}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <ModelPaneIcon name="share" />
+          <ModelPaneDetailLinkText>{t`See it`}</ModelPaneDetailLinkText>
+        </ModelPaneDetailLink>
+      </ModelPaneDetail>
+      <ModelPaneDetail>
+        <ModelPaneIcon name="label" />
+        <ModelPaneDetailText>{t`ID #${question.id()}`}</ModelPaneDetailText>
+      </ModelPaneDetail>
+      <ModelPaneDetail>
+        <ModelPaneIcon name="calendar" />
+        <ModelPaneDetailText>
+          {jt`Last edited ${(
+            <DateTime
+              key="day"
+              unit="day"
+              value={question.lastEditInfo().timestamp}
+            />
+          )}`}
+        </ModelPaneDetailText>
+      </ModelPaneDetail>
+      {table?.fields && (
```

Comment:
@daltojohnso could you please double check and confirm if it's ok to reference fields like that?

Reply:
it should be fine, I think. the `fields` prop is set in the metadata selector for normal tables, and we try to mimic that anywhere we need a "virtual" table

Reply:
we should be able to use it here, so if it isn't fine in some scenario, then we should treat it like a bug.

Reply:
thanks!